content/general/concepts/big-o-notation/big-o-notation.md

```diff
@@ -21,16 +21,16 @@ Here are some common runtimes:
 
 ![Graph of the different kinds of Big-O notation](https://raw.githubusercontent.com/Codecademy/docs/main/media/big-o-graph.png)
 
-|  Runtime   |                Name                | Description                                                                    | Use Case(s)                                                                                                                                                                 |
-| :--------: | :--------------------------------: | ------------------------------------------------------------------------------ | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
-| $O(n^{2})$ | Polynomial/quadratic time (in red) | The time needed is the input size, `n`, multiplied by itself.                  | [Bubble sort](https://www.codecademy.com/learn/sorting-algorithms-java/modules/bubble-sort-java/cheatsheet) has quadratic time complexity.                                  |
-|   $O(n)$   |       Linear time (in blue)        | The time needed is proportional to the input size `n`.                         | Traversing an [array](https://www.codecademy.com/resources/docs/general/data-structures/array) of `n`-size.                                                                 |
-| $O(ln(n))$ |    Logarithmic time (in green)     | The time is a logarithmic function `ln()` of the input size `n`.               | Analyzing the [binary search algorithm](https://www.codecademy.com/learn/fscp-22-search-graph-search-algorithms/modules/wdcp-22-binary-search-and-search-trees/cheatsheet). |
-|   $O(1)$   |     Constant time (in purple)      | If the time needed by the algorithm is the same, regardless of the input size. | Accessing an element in an array.                                                                                                                                           |
+|  Runtime   |                Name                | Description                                                      | Use Case(s)                                                                                                                                                                 |
```

Comment:
Can we add examples for O(n!) and O(2^n)?

O(n!): The traveling salesman problem
O(2^n): The Fibonacci series or knapsack problem both without DP.